packages/cozy-mespapiers-lib/src/constants/papersDefinitions.json

```diff
@@ -2749,6 +2749,57 @@
         }
       ]
     },
+    {
+      "label": "work_quote",
```

Comment:
What is the meaning of "work_quote" ?

Reply:
Les `label` sont toujours des [qualifications existantes](https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/assets/qualifications.json), ici pour "Devis de travaux"